src/script/backport-create-issue

```diff
@@ -100,14 +91,18 @@ def process_resolve_parent_option(a):
         logging.warning("Parent issues with all backports resolved/rejected will be marked Resolved")
 
 def connect_to_redmine(a):
-    if a.key:
-        logging.info("Redmine key was provided; using it")
-        return Redmine(redmine_endpoint, key=a.key)
-    elif a.user and a.password:
-        logging.info("Redmine username and password were provided; using them")
-        return Redmine(redmine_endpoint, username=a.user, password=a.password)
+    full_path=os.path.expanduser(redmine_key_file)
+    if os.path.exists(full_path) and os.path.getsize(full_path) > 0:
+        f=open(full_path, "r")
+        if f.mode == 'r':
+            redmine_key = f.read()
```

Comment:
A more conventional way to do this is:

````
try:
    with open(...) as f:
        redmine_key = f.read()
except FileNotFoundError as e:
    pass
````

Reply:
Done.